lib/platform/git/storage.ts

```diff
@@ -42,6 +42,13 @@ interface LocalConfig extends StorageConfig {
   branchPrefix: string;
 }
 
+export type CommitFile = {
```

Comment:
```suggestion
export type CommitFilesConfig = {
```

References have to be updated too